usr/sbin/pkcscca/pkcscca.mk

```diff
@@ -1,13 +1,51 @@
 sbin_PROGRAMS += usr/sbin/pkcscca/pkcscca
 noinst_HEADERS += usr/sbin/pkcscca/pkcscca.h
+noinst_HEADERS += usr/lib/common/defs.h
+noinst_HEADERS += usr/lib/common/host_defs.h
+noinst_HEADERS += usr/include/local_types.h
+noinst_HEADERS += usr/lib/common/h_extern.h
+noinst_HEADERS += usr/lib/common/pkcs_utils.h
 
-usr_sbin_pkcscca_pkcscca_LDFLAGS = -lcrypto -ldl
+usr_sbin_pkcscca_pkcscca_LDFLAGS = -lcrypto -ldl -lrt
 
 usr_sbin_pkcscca_pkcscca_CFLAGS  =					\
 	-DSTDLL_NAME=\"pkcscca\"					\
-	-I${srcdir}/usr/include -I${srcdir}/usr/lib/common		\
+	-DTOK_NEW_DATA_STORE=0x0003000c					\
+	-DLINUX -DNOCDMF -DNODSA -DNODH -DNOECB				\
+	-DOCK_NO_SET_PERM -DOCK_NO_LOCAL_RNG				\
+	-I${srcdir}/usr/include 					\
+	-I${srcdir}/usr/lib/common					\
 	-I${srcdir}/usr/sbin/pkcscca
 
 usr_sbin_pkcscca_pkcscca_SOURCES =					\
```

Comment:
This definition pulls in all the symbols from these files.  Why not create a convenience library "libcommon.a" that we only use here to link against?

Reply:
Well, yes that's certainly an idea, also because the way it is currently it builds all these source files separately for every token (and now also for the pkcscca tool). 

HOWEVER: the reason that it is that way is probably because each token may (and coes) build these common code parts with different compile options.  Not sure if it is really required to build them with different options, but that's how it is currently. I would not like to change this in this PR, nevertheless, we should try to clean this up someday, compile all the common code parts with the same options, and then create a libcommon.a out of them. This would certainly reduce compile time, and make tings much clearer.